bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -35,12 +35,6 @@
   <![endif]-->
 {% endblock %}
 
-{% block optimizely %}
```

Comment:
Yeah, since this new homepage is for en-US, and we pretty much only test on en-US, I figured it's safe to remove from this (now) non en-US template.